libs/docs/src/docs/no-version/building-a-custom-devfile-registry.md

```diff
@@ -61,7 +61,7 @@ your cloud or cluster to form the devfile registry.
 
 ### Prerequisites
 
-- Golang 1.13.x or higher
+- Golang 1.17.x or higher
```

Comment:
We seem to be missing some context in the original procedure, starting from step 1.  There's mention of running the `./build_image.sh` script but that's located in the `registry-support` repo: https://github.com/devfile/registry-support/tree/main/build-tools.  I think we should mention forking the `devfile/registry-support` repo similar to the previous section `Creating a Git Repository` as the first step.  

We should also make sure the pre-reqs are consistent in the registry-support repo docs too.  They currently say `Golang 1.13.x or higher `

Reply:
> We should also make sure the pre-reqs are consistent in the registry-support repo docs too. They currently say `Golang 1.13.x or higher`

@kim-tsao I agree with keeping the pre-reqs consistent but since the index server is now running on `Golang 1.17.x` shouldn't we require that version being it is the highest version in `registry-support`?

Reply:
yes, I'm suggesting we update the repo docs too with the latest supported version

Reply:
@kim-tsao Ah ok, I'll review the other docs to make sure this is consistent with requiring `Golang 1.17.x or later`.

Reply:
Should really update these feedback changes to the current version we are supporting:
```suggestion
- Golang 1.18.x or higher
```